static/latest-version

```diff
@@ -1 +1 @@
-1.7.1
\ No newline at end of file
+1.8.1
```

Comment:
Looks like there is still an extra newline here?

Reply:
Fixed!